### ISSUE TITLE
Check YouTube Video Integration

### ISSUE THREAD

**Issue by marty-2015** [unikubehq/marketing#75]
@tobifroe can you pleas check if the youtube video integration works now? Last time we tried, Michael relocated the video in a local storage, because the youtube-player didn't start to play on the website.

**Comment by tobifroe** [comment]
@marty-2015 I think there was a misunderstanding - so far, the video component only supports direct urls to video files - eg in our cloud storage.

**Comment by Truthans** [comment]
@tobifroe true. Ideally we want to be able to embed youtube videos in different parts of the website. For example blog posts or in the accordeons on the features page.

**Comment by tobifroe** [comment]
@Truthans @marty-2015 The Text/Video section can now handle YouTube videos.

In order to embed a YT video, click share -> embed and copy the link that's in the code snippet. Formatting should look like this: https://www.youtube.com/embed/IxlbsfuYH8A
Paste it in the video URL field and you should be good to go. Please don't put a trailing slash at the end of the YouTube URL, that might lead to problems.
